patroni/postgresql/bootstrap.py

```diff
@@ -103,6 +103,7 @@ def option_is_allowed(name: str) -> bool:
                     if len(keys) != 1 or not isinstance(opt[keys[0]], str) or not option_is_allowed(keys[0]):
                         error_handler('Error when parsing {0} key-value option {1}: only one key-value is allowed'
                                       ' and value should be a string'.format(tool, opt[keys[0]]))
+                        continue
                     user_options.append('--{0}={1}'.format(keys[0], unquote(opt[keys[0]])))
```

Comment:
I would prefer to get rid of `continue`:
```suggestion
                    if len(keys) == 1 and isinstance(opt[keys[0]], str) and option_is_allowed(keys[0]):
                        user_options.append('--{0}={1}'.format(keys[0], unquote(opt[keys[0]])))
                    else:
                        error_handler('Error when parsing {0} key-value option {1}: only one key-value is allowed'
                                      ' and value should be a string'.format(tool, opt[keys[0]]))
```